Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -8,20 +8,22 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 using UnityEngine.Events;
 using UnityEngine.Serialization;
 
 namespace Microsoft.MixedReality.Toolkit.Experimental.UI
 {
     /// <summary>
-    /// A set of child objects organized in a series of Rows/Columns that can scroll in either the X or Y direction.
-    /// </summary>
+    /// A scrollable frame where content scroll is triggered by manual controller click and drag or according to pagination settings.
+    //// </summary>
+    [ExecuteAlways]
```

Comment:
>[ExecuteAlways] [](start = 4, length = 15)

i assume this is to have the right layout already displayed in edit mode? can you add a comment :)